cfpb/cfgov-refresh
# Email signup style updates

Makes some small tweaks to email signup styles, mostly related to H3 header and inline disclaimer options

## Changes

- Inline disclaimer and button sit on same baseline
- Inset email signup gets 15px left/right padding at 320px & below
- Inline disclaimer display starts above 320px
- On browse pages, inset email takes up 4 of 9 columns above 650px
- Email signup label drops to 16px on mobile (it's currently styled as an H4, but we now have the option for an H3 header on the signup, which drops to H4 on mobile)

## Testing

1. Check out branch
1. Run `gulp build`
2. Create a browse page. Add a full width text module with an inset email that has an inline disclaimer
3. Verify email signup displays as expected across page widths

## Screenshots
### Inline disclaimer/button positioning
<img width="395" alt="screen shot 2018-05-17 at 9 50 40 am" src="https://user-images.githubusercontent.com/778171/40191959-cb3ed444-59b7-11e8-9672-463c2ef2eee3.png">

### Inset email at 320px breakpoint
<img width="306" alt="screen shot 2018-05-17 at 9 46 27 am" src="https://user-images.githubusercontent.com/778171/40191962-cf16f358-59b7-11e8-930f-691ac412a2c9.png">

### Inset email at 4/9 columns on browse pages
<img width="908" alt="screen shot 2018-05-17 at 9 49 44 am" src="https://user-images.githubusercontent.com/778171/40191965-d4116316-59b7-11e8-8b16-621b7035eaf2.png">


## Checklist


## Testing checklist

### Browsers


### Accessibility


### Other
